### PR TITLE
Add CA KRAConnectorService to v2 APIs

### PR DESCRIPTION
**Pull request by fmarco76**

### CODE REVIEW

File: base/ca/src/main/java/org/dogtagpki/server/ca/rest/v2/KRAConnectorServlet.java [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+import com.netscape.certsrv.util.JSONSerializer;
+import com.netscape.cms.servlet.admin.KRAConnectorProcessor;
+
+/**
+ * @author Marco Fargetta {@literal <mfargett@redhat.com>}
+ * @author Ade Lee
+ */
+@WebServlet(
+        name = "caKraConnector",
+        urlPatterns = "/v2/admin/kraconnector/*")
```
Comment (edewata) [inline base/ca/src/main/java/org/dogtagpki/server/ca/rest/v2/KRAConnectorServlet.java:32]: For consistency we probably should use plurals (i.e. `kraconnectors`), but we can do it separately later. I think this is one of the earliest REST services we had so the naming convention wasn't really established.

**Review by edewata — APPROVED** [review]
LGTM

File: base/ca/src/main/java/org/dogtagpki/server/ca/rest/v2/KRAConnectorServlet.java (hunk 1 above)
Comment (fmarco76) [reply base/ca/src/main/java/org/dogtagpki/server/ca/rest/v2/KRAConnectorServlet.java:32]: Yes, we can review all paths and operations when all services are converted

**Comment by fmarco76** [comment]
@edewata Thanks!

**Pull request merged**